inbox/api/wsgi.py

```diff
@@ -28,58 +26,4 @@ def init_process(self) -> None:
 LOGLEVEL = config.get("LOGLEVEL", 10)
 
 
-class JsonRequestFormatter(json_log_formatter.JSONFormatter):
```

Comment:
This is used from the other closed source repository.

Reply:
Restored everything from this file.